zarr/tests/conftest.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+
+@pytest.fixture(scope="session")
+def azurite():
+    import docker
+
+    print("Starting azurite docker container")
+    client = docker.from_env()
+    azurite = client.containers.run(
+        "mcr.microsoft.com/azure-storage/azurite",
+        "azurite-blob --loose --blobHost 0.0.0.0",
+        detach=True,
+        ports={"10000": "10000"},
+    )
+    print("Successfully created azurite container...")
+    yield azurite
+    print("Teardown azurite docker container")
+    azurite.stop()
```

Comment:
Currently this file is what is breaking the tests.

I think a azure blob emulator is already running on localhost port 10000 if you see the Github actions workflow [file](https://github.com/zarr-developers/zarr-python/blob/master/.github/workflows/python-package.yml). So not sure this is needed, you can try and remove it I guess?

Reply:
Thanks. I've updated the things to remove the fixture and just rely on azurite being running. I added the docker command to the contributing docs.